content/posts/2022/kerberos-all-steps-workflow.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Easily enable Kerberos for all the steps of a workflow"
+date: 2022-11-30T14:30:00+01:00
+tags:
+  - kerberos
+---
+
+Up until now, it was only possible to enable Kerberos support for a workflow on
+a step-by-step basis. This changes with REANA 0.9.0, which makes it easy to
+enable Kerberos authentication for an entire workflow by using only a single
+hint in its specification file.
```

Comment:
The phrase "This changes with REANA 0.9.0..." comes perhaps too soon. It disturbs the flow. I would put up an example, and move this phrase down the text a bit, just before the novelty is introduced.

(Also, we can end the phrase with "... only a single hint in the global workflow specification file.")